src/transformers/configuration_transfo_xl.py

```diff
@@ -187,7 +179,9 @@ def __init__(
 
     @property
     def max_position_embeddings(self):
-        return self.tgt_len + self.ext_len + self.mem_len
+        # Message copied from Transformer-XL documentation
+        logger.warning(f"The model {self.model_type} is one of the few models that has no sequence length limit.")
```

Comment:
I would put it under `info` here, users are afraid of warnings and there's nothing for them to worry about here

Reply:
You are right, thanks!